config/default.yaml

```diff
@@ -60,3 +60,7 @@ featureToggles:
 launchDarkly:
   offline: false
 allowedCourts: Swansea Civil Justice Centre
+maxDocumentLimits:
```

Comment:
Does it need to be in config? These things ideally should not change. Already allowed file types & file size is defined in "app/case/definition.ts"